### ISSUE TITLE
How to evaluate and test the data after training

### ISSUE THREAD

**Issue by ZhouAo-ZA** [HeatherJiaZG/SuperGlue-pytorch#9]
After using the COCO data set for training,
1、 how to evaluate and test the accuracy, 
2、the training efficiency is too low. Has anyone tried to modify it to batch training? 
3、The single-machine multi-card merge time reports the dimensionality error. Does anyone try to solve it?

Thanks to HeatherJiaZG for the training code, but unfortunately he doesn't seem to maintain this library anymore. Someone can provide some help, thanks a lot

**Comment by Liukailong9560** [comment]
Hi, Did you solve this problem?

**Comment by sunset326** [comment]
老哥，请问你知道训练结束后如何评估吗

**Comment by Shuhul24** [comment]
I think we can execute the `match_pairs.py` to check the matching (visualizations), but the problem is that when I run the `match_path.py` file, the error comes to be
```python
Will write matches to directory "dump_match_pairs"
[ WARN:0@1.733] global loadsave.cpp:248 findDecoder imread_('/home/shuhulh/assets/scannet_sample_images/scene0711_00_frame-001680.jpg'): can't open/read file: check file path/integrity
[ WARN:0@1.733] global loadsave.cpp:248 findDecoder imread_('/home/shuhulh/assets/scannet_sample_images/scene0711_00_frame-001995.jpg'): can't open/read file: check file path/integrity
Problem reading image pair: /home/shuhulh/assets/scannet_sample_images/scene0711_00_frame-001680.jpg /home/shuhulh/assets/scannet_sample_images/scene0711_00_frame-001995.jpg
```
Is there any way I can check how this can be avoided?
